lib/stripe_mock/request_handlers/subscriptions.rb

```diff
@@ -102,7 +102,7 @@ def create_subscription(route, method_url, params, headers)
           customer[:default_source] = new_card[:id]
         end
 
-        allowed_params = %w(customer application_fee_percent coupon items metadata plan quantity source tax_percent trial_end trial_period_days current_period_start created prorate billing_cycle_anchor billing days_until_due idempotency_key enable_incomplete_payments cancel_at_period_end default_tax_rates payment_behavior pending_invoice_item_interval default_payment_method collection_method)
+        allowed_params = %w(customer application_fee_percent coupon items metadata plan quantity source tax_percent trial_end trial_period_days current_period_start created prorate billing_cycle_anchor billing days_until_due idempotency_key enable_incomplete_payments cancel_at_period_end default_tax_rates payment_behavior pending_invoice_item_interval default_payment_method collection_method on_behalf_of transfer_data)
```

Comment:
Do you mean to have added `transfer_data` in here?
It's not referenced anywhere else

Reply:
Yes. I'm just not sure yet how to add specs. Transfer_data is for the destination charges, so we need to add it. I Just don't know yet what I want to do to write meaningful specs.

Reply:
Thinking about it, I think I'll just open the field as I did with on_behalf_of. I'll add that.